src/simple_cache_client.rs

```diff
@@ -41,13 +41,60 @@ where
     }
 }
 
+/// Policy controlling how the item TTL is updated when it is updated.
+#[derive(Copy, Clone, Debug)]
+pub struct TtlPolicy {
```

Comment:
This is going to need to be `CollectionTtl` for consistency https://github.com/momentohq/client-sdk-dotnet-incubating/blob/main/src/Momento.Sdk.Incubating/Requests/CollectionTtl.cs#L26

You might consider reusing some of the documentation in there too ❤️